controllers/droneController.js

```diff
@@ -0,0 +1,38 @@
+const Drone = require("./../models/Drone.model")
+
+
+exports.getAllDrones = async(req, res) => {
+
+    const allDrones = await Drone.find({})
+
+    console.log(allDrones)
+
+    res.render("drones/list", {
+        data: allDrones
+    })
+}
+
+exports.viewCreateDrone = async(req, res) => {
+
+    res.render("drones/create")
```

Comment:
Aca a tu render le hace falta el nombre del documento hbs  es que es create-form no create
```suggestion
 res.render("drones/create-form");
 ```